core/src/main/java/com/klaviyo/coresdk/networking/KlaviyoRequest.kt

```diff
@@ -0,0 +1,22 @@
+package com.klaviyo.coresdk.networking
+
+abstract class KlaviyoRequest: NetworkRequest() {
+    companion object {
+        internal const val BASE_URL = "https://a.klaviyo.com"
```

Comment:
I wanna move this into some config value somewhere so that different build flavours can point to different endpoints (like a localhost) but I won't tackle that in this initial PR.